Tuturial-Structure-Template.md

```diff
@@ -1,68 +1,178 @@
-# Title #
-[header image]
+>The title describes a task that you want to do. For example: **Implement step-up authentication**.
+> 
+> To write a page, use the style and tone suggest in the style guide.
 
-## Intro ##
-> (do not include section title) 
+# Title
```

Comment:
Please clearly define the application scope of this template.

Reply:
I think this template applies to all docs aiming at finishing a task. So that would be for our current quickstart, basic features, and advanced features. Some of the troubleshooting in FAQ also fit this category.

Reply:
Any procedure.
Overview
release notes
procedure
concept

Console will be redesigned -- to be checked with Jun.
Agora Analytics